.github/workflows/conformance-datapath.yaml

```diff
@@ -121,40 +121,56 @@ jobs:
       matrix:
         include:
           # See https://github.com/cilium/cilium/issues/20606 for configuration table
-          - kernel: '4.19-main'
+          - name: '1'
+            kernel: '4.19-main'
             kube-proxy: 'iptables'
             kpr: 'disabled'
             tunnel: 'vxlan'
             encryption: 'ipsec'
 
-          - kernel: '5.4-main'
+          - name: '2'
+            kernel: '5.4-main'
             kube-proxy: 'iptables'
             kpr: 'disabled'
             tunnel: 'disabled'
             encryption: 'ipsec'
 
-          - kernel: '5.10-main'
+          - name: '3'
+            kernel: '5.15-main'
+            kube-proxy: 'iptables'
+            kpr: 'disabled'
+            tunnel: 'disabled'
+            encryption: 'ipsec'
+            endpoint-routes: 'true'
```

Comment:
Not ordered by kernel anymore?

The commit description could probably use an update to explain why we have twice the same kernel and to update "We enable it only on two jobs".

Reply:
Ordered and added.